keps/sig-auth/3130-kms-observability/README.md

```diff
@@ -0,0 +1,224 @@
+# KEP-3130: KMS Observability
+
+<!-- toc -->
+- [Release Signoff Checklist](#release-signoff-checklist)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+- [Production Readiness Review Questionnaire](#production-readiness-review-questionnaire)
+  - [Feature Enablement and Rollback](#feature-enablement-and-rollback)
+  - [Monitoring Requirements](#monitoring-requirements)
+  - [Dependencies](#dependencies)
+  - [Scalability](#scalability)
+  - [Troubleshooting](#troubleshooting)
+- [Implementation History](#implementation-history)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
+  - [ ] e2e Tests for all Beta API Operations (endpoints)
+  - [ ] (R) Ensure GA e2e tests for meet requirements for [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [ ] (R) Graduation criteria is in place
```

Comment:
With regard to Graduation criteria, since we are going with the option of generating a UID for the Alpha version of this KEP. We could have a criteria to switch to audit ID as a graduation criteria to go Beta if this is the direction we want to go with in the future.
Or we could phrase that in a more generic way such as "integrating with existing audit identification mechanisms".

Reply:
We're still investigating if integration with audit ID is possible but I'm not sure if we want to add that here as graduation criteria before knowing it would be feasible. If we can edit the graduation criteria at a later point then we can add it but if not, we should update it after investigation.

Reply:
In addition to investigating if we can use audit ID, there's also the concern that not all calls to the KMS correspond to an audit log, which would make audit ID not ideal.

Reply:
Makes sense